tests/core/models/test_work.py

```diff
@@ -1145,7 +1145,12 @@ def assert_time_match(python, postgres):
         assert edition.publisher == search_doc["publisher"]
         assert edition.imprint == search_doc["imprint"]
         assert edition.permanent_work_id == search_doc["permanent_work_id"]
-        assert edition.published == datetime.date.fromtimestamp(search_doc["published"])
+        assert (
+            edition.published
+            == datetime.datetime.fromtimestamp(
+                search_doc["published"], tz=pytz.UTC
```

Comment:
There already existsts `from_timestamp` helper function implemented [here](https://github.com/ThePalaceProject/circulation/blob/main/core/util/datetime_helpers.py#L20-L25) that does the same thing and it is [already imported](https://github.com/ThePalaceProject/circulation/blob/main/tests/core/models/test_work.py#L23) in this test file so maybe it is better to use that?

Reply:
@tdilauro Oh, I see you already mentioned `from_timestamp` to Mark so you are aware of `from_timestamp`, is there a reason you didn't use that?